Problem2.rb

```diff
@@ -0,0 +1,32 @@
+require 'rspec'
+
+def find_max_numbers(array)
+
+  first = 0
+  second = 0
+
+  array.each do |x|
+
+    if x > first
+      second = first
+      first = x
+    elsif x > second
+      second = x
+    end
+
+  end
+
+  return Array.new([first, second])

```

Comment:
Just for reference: https://github.com/bbatsov/ruby-style-guide
Every major programming language has it's own.